test/api/api.js

```diff
@@ -1,212 +1,231 @@
-q.module('Player API');
-
-test('videojs should exist on the window', function() {
-  ok(window.videojs, 'videojs exists on the window');
+QUnit.module('Player API');
+QUnit.test('videojs should exist on the window', function() {
+  QUnit.ok(window.videojs, 'videojs exists on the window');
 });
 
-test('should be able to access expected player API methods', function() {
-  var player = videojs.getComponent('Player').prototype;
+QUnit.test('should be able to access expected player API methods', function() {
+  const player = videojs.getComponent('Player').prototype;
 
   // Native HTML5 Methods
-  ok(player.error, 'error exists');
-  ok(player.src, 'src exists');
-  ok(player.currentSrc, 'currentSrc exists');
-  ok(player.buffered, 'buffered exists');
-  ok(player.load, 'load exists');
-  ok(player.seeking, 'seeking exists');
-  ok(player.currentTime, 'currentTime exists');
-  ok(player.duration, 'duration exists');
-  ok(player.paused, 'paused exists');
-  ok(player.ended, 'ended exists');
-  ok(player.autoplay, 'autoplay exists');
-  ok(player.loop, 'loop exists');
-  ok(player.play , 'play exists');
-  ok(player.pause , 'pause exists');
-  ok(player.controls, 'controls exists');
-  ok(player.volume, 'volume exists');
-  ok(player.muted, 'muted exists');
-  ok(player.width, 'width exists');
-  ok(player.height, 'height exists');
-  ok(player.poster, 'poster exists');
-  ok(player.textTracks, 'textTracks exists');
-  ok(player.requestFullscreen, 'requestFullscreen exists');
-  ok(player.exitFullscreen, 'exitFullscreen exists');
-  ok(player.playbackRate, 'playbackRate exists');
-  ok(player.networkState, 'networkState exists');
-  ok(player.readyState, 'readyState exists');
+  QUnit.ok(player.error, 'error exists');
+  QUnit.ok(player.src, 'src exists');
+  QUnit.ok(player.currentSrc, 'currentSrc exists');
+  QUnit.ok(player.buffered, 'buffered exists');
+  QUnit.ok(player.load, 'load exists');
+  QUnit.ok(player.seeking, 'seeking exists');
+  QUnit.ok(player.currentTime, 'currentTime exists');
+  QUnit.ok(player.duration, 'duration exists');
+  QUnit.ok(player.paused, 'paused exists');
+  QUnit.ok(player.ended, 'ended exists');
+  QUnit.ok(player.autoplay, 'autoplay exists');
+  QUnit.ok(player.loop, 'loop exists');
+  QUnit.ok(player.play, 'play exists');
+  QUnit.ok(player.pause, 'pause exists');
+  QUnit.ok(player.controls, 'controls exists');
+  QUnit.ok(player.volume, 'volume exists');
+  QUnit.ok(player.muted, 'muted exists');
+  QUnit.ok(player.width, 'width exists');
+  QUnit.ok(player.height, 'height exists');
+  QUnit.ok(player.poster, 'poster exists');
+  QUnit.ok(player.textTracks, 'textTracks exists');
+  QUnit.ok(player.requestFullscreen, 'requestFullscreen exists');
+  QUnit.ok(player.exitFullscreen, 'exitFullscreen exists');
+  QUnit.ok(player.playbackRate, 'playbackRate exists');
+  QUnit.ok(player.networkState, 'networkState exists');
+  QUnit.ok(player.readyState, 'readyState exists');
 
   // Unsupported Native HTML5 Methods
```

Comment:
most of these are available now. Though, we shouldn't turn it on as part of this.